.github/workflows/golangci-lint.yaml

```diff
@@ -22,8 +22,8 @@ jobs:
         run: ./openstack-k8s-operators-ci/test-runner/govet.sh
       - name: Run golint.sh
         run: ./openstack-k8s-operators-ci/test-runner/golint.sh
-      - name: Run gotest.sh
-        run: ./openstack-k8s-operators-ci/test-runner/gotest.sh
+      - name: Run gotest
+        run: make gotest
```

Comment:
eventually this whole job should be moved to prow. But until that this is an easy way to run the tests.